src/main/java/bio/terra/cli/command/resource/create/AwsSageMakerNotebook.java

```diff
@@ -70,10 +70,9 @@ protected void execute() {
             .instanceType(instanceType)
             .region(region);
 
-    bio.terra.cli.businessobject.resource.AwsSageMakerNotebook createdResource =
-        bio.terra.cli.businessobject.resource.AwsSageMakerNotebook.createControlled(
-            createParams.build());
-    formatOption.printReturnValue(
-        new UFAwsSageMakerNotebook(createdResource), AwsSageMakerNotebook::printText);
+    bio.terra.cli.businessobject.resource.AwsSageMakerNotebook.createControlled(
```

Comment:
Just curious, why couldn't we just import AwsSageMakerNotebook?

Reply:
the command class itself is called AwsSageMakerNotebook, hence importing the resource might lead to conflicting names. Most IDEs and compilers are likely to interpret it as the command class and not the resource class. Resource and command classes could have been differently named to avoid this (example: PD resource and UF resource), however i kept with the existing design of same command-businessObject names. 
This full name spec mostly comes up only in few commands, so changes to rename them would be minimal impact and low priority